.github/workflows/system-test-notification.yml

```diff
@@ -0,0 +1,32 @@
+name: system-test-notification
+
+on:
+  # This needs to be `pull_request_target` instead of `pull_request`
+  # to be able to write a comment on the pull request.
+  pull_request_target:
+    types:
+      - opened
```

Comment:
a ` pull_request:` workflow can write comments in PRs, what gives it permissions is the `permisions` setting

Reply:
Let me verify this again. But yesterday, when I tried it - with the `permissions` already set, I got  a `403` error.

Reply:
See this [PR](https://github.com/unelastisch/actions-playground/pull/20) in my personal repository.

The notification is failing with a `403` error in this [workflow run](https://github.com/unelastisch/actions-playground/actions/runs/5541288320/jobs/10114495237?pr=20).

Reply:
I think it's because the github token does not have any write permissions, even if you set it expliclity on a PR from a forked repo.

Reply:
since the wf does not make any checkout from the PR, I think is fine